lxd/instance/drivers/driver_lxc.go

```diff
@@ -2564,6 +2564,19 @@ func (d *lxc) onStart(_ map[string]string) error {
 	if d.expandedConfig["limits.network.priority"] != "" {
 		go func(d *lxc) {
 			d.fromHook = false
+
+			timeout := time.Now().Add(10 * time.Second)
```

Comment:
@mihalicyn can you check whether we can avoid needing this loop by instead changing the check in `setNetworkPriority` to use `if c.InitPID() <= 0 {` instead, like we do in `deviceTaskBalance`?

Reply:
great idea! Have done.